cloud_functions/climateiq_export_to_aws_cf/main.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+import os
+
+import boto3
+import flask
+import functions_framework
+from google.cloud import secretmanager
+from google.cloud.storage import client as gcs_client
+
+# GCS bucket name, where merged prediction outputs are stored.
+GCS_BUCKET_NAME = (
+    os.environ.get("BUCKET_PREFIX", "") + "climateiq-spatialized-merged-predictions"
+)
+# AWS bucket name, where to copy files to.
+S3_BUCKET_NAME = "climateiq-data-delivery"
+
+# IDs for retrieving secrets to authenticate to AWS
+PROJECT_ID = "climateiq"
+AWS_ACCESS_KEY_ID = "climasens-aws-access-key-id"
+AWS_SECRET_ACCESS_KEY = "climasens-aws-secret-access-key"
+
+
+@functions_framework.http
+def export_to_aws(request: flask.Request) -> tuple[str, int]:
+    try:
+        prefix = _get_prefix_id(request)
+    except ValueError as e:
+        return (str(e), 400)
+
+    storage_client = gcs_client.Client()
+    blobs_to_export = storage_client.list_blobs(GCS_BUCKET_NAME, prefix=prefix)
+
+    if not len(blobs_to_export):
+        return (f"No blobs found with prefix {prefix}", 200)
```

Comment:
hmm lets return a 400 error here too so people are aware no data was actually sent to ClimaSens